tests/components/lacrosse_view/test_init.py

```diff
@@ -92,7 +92,7 @@ async def test_new_token(hass: HomeAssistant) -> None:
     assert len(entries) == 1
     assert entries[0].state == ConfigEntryState.LOADED
 
-    one_hour_after = datetime.utcnow() + timedelta(hours=1)
+    one_hour_after = datetime.now() + timedelta(hours=1)
```

Comment:
I think we should use `dt_util.utcnow` here.

Reply:
I looked at the integration and this case is testing the token refresh which uses a naive time as reference point. Normally when testing integrations including coordinators the reference time is an aware UTC time. We should match the reference time in the integration so I think it's ok like this.